IS document availability in SFX

ashu720it15 opened the pull request:
Adding changes to make IS Live documents available in the SFX to debug at runtime. It will help in reduce the TTM and TTR of IS related ICMs.
<img width="1487" height="1228" alt="image (2)" src="https://github.com/user-attachments/assets/f156388c-decd-4902-8467-5c2d7f59d9ec" />

chensation commented on src/SfxWeb/src/app/Models/RawDataTypes.ts:
```diff
@@ -1098,6 +1098,102 @@ export interface IRawRepairTask {
         ResultDetails?: string;
     }
 
+export interface IDoc {
```
what is IDoc? This is a very vague name. RawDataTypes.ts contains all raw datatypes from SF. If you are using it for InfrastructureDoc, please follow the convention established here and name this accordingly as IRawInfratructureDoc. The same applies for the other interfaces you added here, such as IJob, or the ones without the I in front to indicate it is an interface, such as JobSetsRequiringApproval

chensation commented on src/SfxWeb/src/app/views/cluster/base/base.component.ts:
```diff
@@ -78,7 +78,7 @@ export class BaseComponent implements OnInit, IBaseView {
       if (this.dataService.clusterManifest.isRepairManagerEnabled) {
         this.tabs = this.tabs.concat({
-          name: 'infrastructure jobs',
+          name: 'infrastructure jobs and documents',
```
Is appending "and documents" to the title needed? This extends the tab length by quite a bit without much gain

ashu720it15 commented:
> 1. Please add a description and some screen shot to the PR to explain the purpose of the changes.
> 2. Please address comments from me and copilot.
> 3. **Please add cypress tests for the scenario you are adding**

description added also added the screenshot and test also added.

chensation commented on src/SfxWeb/src/app/Models/RawDataTypes.ts:
```diff
@@ -1098,6 +1098,101 @@ export interface IRawRepairTask {
+        Jobset: string[];
+    }
+
+export interface IJob {
```
These still need to be clarified and renamed. IJob can be renamed to IRawInfrastructureDocJob. Same for the ones below.
